Solution1/ConsoleApp/AutoMap/WeatherProfile.cs

```diff
@@ -0,0 +1,37 @@
+﻿using DataAccessLayer.Model;
+using BusinessLayer.DTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConsoleApp.AutoMap
+{
+    public class WeatherProfile : AutoMapper.Profile 
+    {
+        public WeatherProfile()
+        {
+            CreateMap<Weather, WeatherDTO>()
+                .ForMember(dest => dest.CityName, conf => conf.MapFrom(src => src.CityName))
+                .ForMember(dest => dest.Temp, conf => conf.MapFrom(src => src.Main.Temp))
+                .ForMember(dest => dest.Comment, conf => conf.MapFrom(src => GetCommentByTemp(src.Main.Temp)));
+        }
+
+        string GetCommentByTemp(double temp)
+        {
+            switch (temp)
```

Comment:
try to use https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/switch-expression

Reply:
+